grails-app/views/wps/show.gsp

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="${resource(dir: 'css', file: 'imosTheme.css')}" type="text/css">
+    </head>
+    <body>
+        <div class="imosHeader">
+            <div class="container">
+                <a class="btn" role="button" href="https://imos.aodn.org.au/imos123/home">
+                    <img src="https://static.emii.org.au/images/logo/IMOS-Ocean-Portal-logo.png" alt="IMOS logo">
+                </a>
+            </div>
+        </div>
+        <div class="container">
+            <h2><g:message code="job.status" default="WPS Status"/></h2>
+            <dl>
+                <g:labelledContent labelCode="job.id.label">${job.uuid}</g:labelledContent>
+                <g:labelledContent labelCode="job.createdTimestamp.label" if="${job.createdTimestamp}" >
```

Comment:
If the job is finished then I wouldn't want to see 'Submitted:' header at all


![screen shot 2015-09-24 at 1 54 56 pm](https://cloud.githubusercontent.com/assets/1066981/10065085/e926ccf2-62c3-11e5-889e-81c921b8c57d.png)